packages/wonder-blocks-typography/src/util/styles.ts

```diff
@@ -36,6 +36,7 @@ const styles: StyleDeclaration = StyleSheet.create({
     BodyText: {
         ...common,
         fontFamily: font.family.sans,
+        margin: 0,
```

Comment:
Should we do the same for the `Heading` styles as well?

Reply:
Good question! I thought about it but decided to keep the Heading styles the same since the original components didn't zero out margins. @jandrade what do you think?